core/conf/demo/server_config.yaml

```diff
@@ -165,3 +168,24 @@ gpu_resource_config:
 #----------------------+------------------------------------------------------------+------------+-----------------+
 tracing_config:
   json_config_path:
+
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# WAL Config           | Description                                                | Type       | Default         |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# enable               | Switch of function WAL.                                    | Boolean    | false           |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# recovery_error_ignore| Whether ignore the error which happens during WAL recovery | Boolean    | true            |
+#                      | stage.                                                     |            |                 |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# buffer_size          | The size of the WAL buffer. Unit is MB.                    | Integer    | 256 (MB)        |
```

Comment:
Sum total of the read buffer and the write buffer in MBs. `buffer_size` must be in range [64, 4096] (MB) . If the value you specified is out of range, Milvus automatically uses the boundary value closest to the specified value. It is recommended you set `buffer_size` to a value greater than the inserted data size of a single insert operation for better performance.

Reply:
Please apply this change to other files.